vector/src/main/java/im/vector/app/features/roomdirectory/picker/RoomDirectoryPickerFragment.kt

```diff
@@ -65,6 +66,11 @@ class RoomDirectoryPickerFragment @Inject constructor(val roomDirectoryPickerVie
 
         sharedActionViewModel = activityViewModelProvider.get(RoomDirectorySharedActionViewModel::class.java)
         setupRecyclerView()
+
+        // Give the current data to our controller. There maybe a better way to do that...
+        withState(viewModel) {
+            roomDirectoryPickerController.currentRoomDirectoryData = it.roomDirectoryData
```

Comment:
shouldn't this somehow trigger a rebuild on the controller?

Reply:
This will not force a rebuild, but there has been no pb with this code, the controller will build data later.